fluent-react/test/localized_valid.test.js

```diff
@@ -64,17 +79,23 @@ describe("Localized - validation", () => {
     expect(console.error).toHaveBeenCalled();
   });
 
-  test("with single children inside an array", () => {
+  test("is valid to have a children array with a single element inside", () => {
```

Comment:
I updated the examples to include valid configurations. Before the configurations would "just work" even when misconfigured.